### ISSUE TITLE
Cannot install for py3.11

### ISSUE THREAD

**Issue by chhu** [FGlazov/Python-rANSCoder#2]
Hi, could you please update the numba requirements? The version in setup.py is not available for 3.11, a newer version is.

Thanks, great project!

**Comment by FGlazov** [comment]
Hey, thanks for the heads-up. I've made the numba requirements more permissive in 1.0.5 (>= 0.56.3 instead of == 0.56.3) - it should now work on Python 3.11, although I haven't tested it. Feel free to reopen the issue if it that change hasn't fixed the problem, I'll take another look at it then.

The reason it was on a specific version is that the code uses a feature which is tagged as experimental in numba (I use the functionality which compiles classes) - but the API hasn't changed there since I added in the numba dependency, so I used it to hedge against potential depecrations breaking the Repo.

So basically, the Repo may break in the future - but it seems the API has been stable enough for the last year or so, so I'll just fix it when it breaks then.
